src/lib/validationRules/rightAuthenticationMethod/rightAuthenticationMethod.ts

```diff
@@ -25,7 +25,7 @@ const rightAuthenticationMethod: ValidationRule = {
     type: "local",
     name: "Field `token_endpoint_auth_method` must be `none`.",
     description:
-      "Solid OIDC connect does only support `token_endpoint_auth_method` set to `none`. This must be made explicit to comply with the OIDC specification.",
+      "Authentication with Client Identifier Documents is only supported with `token_endpoint_auth_method` set to `none`. This must be made explicit to comply with the Solid OIDC specification.",
```

Comment:
You might want to add a note that not all Solid servers are strict about this compliance point — ESS currently isn't.

Reply:
changed with https://github.com/inrupt/client-identifier-helper/pull/133/commits/25a2e821818647566be8a36114feff15d18f5fd2